dotnet/samples/KernelSyntaxExamples/Example16_CustomLLM.cs

```diff
@@ -3,130 +3,109 @@
+ * no actual model.
  *
- * This might be useful in a few scenarios, for example:
+ * Using a a custom text generation model within SK can be useful in a few scenarios, for example:
  * - You are not using OpenAI or Azure OpenAI models
- * - You are using OpenAI/Azure OpenAI models but the models are behind a web service with a different API schema
  * - You want to use a local model
  *
- * Note that all text generation models are deprecated by OpenAI and will be removed in a future release.
+ * Note that all OpenAI text generation models are deprecated and no longer available to new customers.
  *
  * Refer to example 33 for streaming chat completion.
  */
-// ReSharper disable StringLiteralTypo
 // ReSharper disable once InconsistentNaming
 public static class Example16_CustomLLM
 {
-    private const string LLMResultText = @" ..output from your custom model... Example:
-    AI is awesome because it can help us solve complex problems, enhance our creativity,
-    and improve our lives in many ways. AI can perform tasks that are too difficult,
-    tedious, or dangerous for humans, such as diagnosing diseases, detecting fraud, or
-    exploring space. AI can also augment our abilities and inspire us to create new forms
-    of art, music, or literature. AI can also improve our well-being and happiness by
-    providing personalized recommendations, entertainment, and assistance. AI is awesome";
-
     public static async Task RunAsync()
     {
         await CustomTextGenerationWithSKFunctionAsync();
-
         await CustomTextGenerationAsync();
         await CustomTextGenerationStreamAsync();
     }
 
     private static async Task CustomTextGenerationWithSKFunctionAsync()
     {
-        Console.WriteLine("======== Custom LLM - Text Completion - SKFunction ========");
+        Console.WriteLine("\n======== Custom LLM - Text Completion - SKFunction ========");
 
         KernelBuilder builder = new();
-        builder.Services.AddSingleton(ConsoleLogger.LoggerFactory);
         // Add your text generation service as a singleton instance
         builder.Services.AddKeyedSingleton<ITextGenerationService>("myService1", new MyTextGenerationService());
         // Add your text generation service as a factory method
         builder.Services.AddKeyedSingleton<ITextGenerationService>("myService2", (_, _) => new MyTextGenerationService());
         Kernel kernel = builder.Build();
 
-        const string FunctionDefinition = "Does the text contain grammar errors (Y/N)? Text: {{$input}}";
+        const string FunctionDefinition = "Write one paragraph on {{$input}}";
+        var ParagraphWritingFunction = kernel.CreateFunctionFromPrompt(FunctionDefinition);
 
-        var textValidationFunction = kernel.CreateFunctionFromPrompt(FunctionDefinition);
+        const string Input = "Why AI is awesome";
+        Console.WriteLine($"Function input: {Input}\n");
+        var result = await ParagraphWritingFunction.InvokeAsync(kernel, new());
```

Comment:
Looks like `Input` is not used for function execution:
```suggestion
        var result = await ParagraphWritingFunction.InvokeAsync(kernel, new(Input));
```

Reply:
Indeed! Thanks for catching this,